### PR TITLE
Update Outlook SSO sample to ASP.NET Core latest

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                               |
| New feature?    | no                                |
| New sample?     | no                                |
| Related issues? | none |

## What's in this Pull Request?

Update project to ASP.NET Core latest.

### CODE REVIEW

**Review by AlexJerabek — COMMENTED** [review]
Everything was working fine right up until the end. Screenshots in the comments.

File: Samples/auth/Outlook-Add-in-SSO/README.md [hunk 1]
```diff
@@ -146,61 +94,51 @@ The browser will attempt to redirect back to your app, which may not be running.
...
-![Screen shot of the elipses button in Outlook on the web](buttons-outlook-web.png)
-    * If you're in Outlook on Windows or Mac: On the **Home** tab, select **Choose attachments**. Note that if the Outlook app window is too small, that **Choose attachments** will instead be located on the **Home** tab's **...** (**More commands**) button.<br>
-![Screen shot of the Choose attachments button on Home tab in Outlook on the web](buttons-outlook-desktop.png)
-    
-1. In the **AttachmentDemoWeb** task pane that opens, select the attachments you want to save.
 
-1. Choose **Save selected**
+    - If you're in Outlook on the web: Select the **...** (**More actions**) drop down menu, and then choose **AttachmentDemoWeb**.
+
+    ![Screen shot of the elipses button in Outlook on the web](buttons-outlook-web.png)
```
Comment (AlexJerabek) [inline Samples/auth/Outlook-Add-in-SSO/README.md:114]: This screenshot isn't loading in the PR.
image

**Review by AlexJerabek — APPROVED** [review]
Approving with one missing image note. I also missed the list of Graph permissions my first time through. Take that however you want :-)

File: Samples/auth/Outlook-Add-in-SSO/README.md (hunk 1 above)
Comment (davidchesnut) [reply Samples/auth/Outlook-Add-in-SSO/README.md]: Should be fixed.

**Pull request merged**